City-of-Helsinki/drupal-hdbt
# Uhf 5988 unify tags

# [UHF-5988](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5988)

Tags in different places have different styles and are not unified

## What was done

Created templates for tags and tag list that can be reused in places where needed. Changed styled according to the design.

## How to install

* Make sure your instance is up and running on latest dev branch. (you'll need etusivu, rekry and sote)
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-5988-unify-tags`
* Run `make drush-cr`

## How to test

* In etusivu-instance go to some news article page, the tags should be now replaced with the new styles
* In rekry, go to some job listing page and check that the tags are now with the new style
    * You need to checkout the branch in rekry-intance in order to test this.
    * https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/26
* In sote go to some tpr unit page that offers Swedish service (i.ex. https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/terveydenhoito/hammashoito/hammashoitolat/munkkiniemen-terveysaseman-hammashoitola), the tag should now be with the new styles
* Check that the tag also has margin in the listing https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/terveydenhoito/terveysasemat


## Designers review


* [x] This PR has been visually reviewed by a designer (Sebastian Högnabba)

## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/26
